env/src/main/scala/tofu/env/EnvTraversing.scala

```diff
@@ -1,66 +1,64 @@
 package tofu.env
 
 import monix.eval.Task
-
-import scala.collection.TraversableLike
-import scala.collection.generic.CanBuildFrom
+import monix.execution.compat.BuildFrom
```

Comment:
I think we should use official `scala.collection.compat.BuildFrom` instead

Reply:
I can't see how we might use it having monix API is built upon that

Reply:
I'll try

Reply:
No. Scala 2.12.9 wont build

Reply:
Okay, sad though